aws/scripts/sh/credentials.sh

```
@@ -1,16 +1,37 @@
-mv config ~/.aws/config
-mv credentials ~/.aws/credentials
-export AWS_PROFILE=terraform
+#!/bin/bash
+
+if ! touch config credentials; then
+    echo "Error: Failed to create temporary files" >&2
+    exit 1
+fi
+
+# Validate AWS credentials extraction
+if ! access_key_id=$(echo "$aws_iam_output" | jq -r '.["Credentials"] | .["AccessKeyId"]') || \
```

Shellcheck

[warning] aws_iam_output is referenced but not assigned.

(SC2154)